task-02-damianti/6.3.long_cat_is_cat.py

```diff
@@ -0,0 +1,23 @@
+def count_words(txt):
+    """
+
+    :param txt: (string) The input string to be processed.
+    :return lengths: (dict) A dictionary mapping each word in the input string to its length.
+    """
+    clean_text = ''.join(c for c in txt if c.isalpha() or c.isspace())
+
+    lengths = {word: len(word) for word in clean_text.split()}
+
+    return lengths
+
+
+if __name__ == "__main__":
```

Comment:
Better practice to call underneath the if statement to a main function and there write the wanted code.